README.md

```diff
@@ -30,7 +31,7 @@ Land use permitting analysis for Department of City Planning
     ├── requirements.txt         <- The requirements file for reproducing the analysis environment, e.g.
     │                               generated with `pip freeze > requirements.txt`
     │
-    ├── setup.py                 <- makes project pip installable (pip install -e .) so src can be imported
+    ├── setup.py                 <- Makes project pip installable (pip install -e .) so src can be imported
```

Comment:
Should have caught this earlier. This `setup.py` is now for `laplan`, and the scripts in `src` are not installed/importable this way any more.